modules/configuring-a-provisioning-resource-to-scale-the-cluster.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-expanding-a-cluster-with-on-premise-bare-metal-nodes.adoc
+:_content-type: PROCEDURE
+
+[id="configuring-a-provisioning-resource-to-scale-the-cluster_{context}"]
+= Configuring a provisioning resource to scale the cluster
+
+Create a `Provisioning` custom resource (CR) to enable Metal platform components on the cluster. 
+
+.Procedure
+
+. Create a `Provisioning` CR.
+
+.. Save the following YAML in the `provisioning.yaml` file:
++
+[source,yaml]
+----
+apiVersion: metal3.io/v1alpha1
+kind: Provisioning
+metadata:
+  name: provisioning-configuration
+spec:
+  provisioningNetwork: "Disabled"
+  watchAllNamespaces: false
```

Comment:
Hi @johnwilkins, unlike UPI procedure, the parameter `watchAllNamespaces` needs to be `true` here. The rest LGTM, thks.

Reply:
same for me